server/publicapi/items/service.py

```diff
@@ -228,6 +241,129 @@ def _create_date_range_filter(self, start_date, end_date):
 
         return date_filter
 
+    def _set_fields_filter(self, req):
```

Comment:
just wondering why are all functions item service methods, seems like those could be defined outside and it would make it easier to reuse the code